docs/admin/30-configuration/05-environment-variables.md

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_position: 1
+id: env-variables
+title: Environment Variables
+draft: true
+---
+
+## Environment Variables Overview
```

Comment:
Please give a practical example how these variables can get consumed by OpenCloud. afaik that is:
- CLI parameter
- .env file
- compose file
correct?